### PR TITLE
Disable SLMSnapshotBlockingIntegTests#testSnapshotInProgress

### PR DESCRIPTION
**Pull request by jimczi**
This test fails constantly in master and prs. 

Relates #47689

### CODE REVIEW

File: x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java [hunk 1]
```diff
@@ -85,6 +86,7 @@ public void resetSLMSettings() throws Exception {
         return Arrays.asList(MockRepository.Plugin.class, LocalStateCompositeXPackPlugin.class, IndexLifecycle.class);
     }
 
+    @Ignore("https://github.com/elastic/elasticsearch/issues/47689")
```
Comment (ywelsch) [inline x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java]: can you use `@AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/47689")` here instead?

**Review by ywelsch — CHANGES_REQUESTED** [review]

File: x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java (hunk 1 above)
Comment (jimczi) [reply x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java]: done, thanks

**Comment by jimczi** [comment]
@elasticmachine update branch

**Review by ywelsch — APPROVED** [review]
LGTM

**Pull request merged**